uxarray/grid/lines.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+
+from uxarray.grid.coordinates import node_xyz_to_lonlat_rad
+
+from uxarray.constants import ERROR_TOLERANCE
+
+
+def convert_to_list_if_needed(obj):
```

Comment:
I would make this an internal function and consider a different name.
```suggestion
def _to_list(obj):
```